install.zasilkovna.php

```diff
@@ -425,7 +462,7 @@ private function removeAdministratorFiles() {
         $vm_admin_path = JPATH_ADMINISTRATOR . DS . 'components' . DS . 'com_virtuemart';
         recurse_delete($vm_admin_path . DS . 'models' . DS . 'zasilkovna.php');
         recurse_delete($vm_admin_path . DS . 'models' . DS . 'zasilkovna_orders.php');
-        recurse_delete($vm_admin_path . DS . 'models' . DS . 'zasilkovna_src' . DS);
+        recurse_delete($vm_admin_path . DS . 'models' . DS . 'zasilkovna_src' . DS, true);
```

Comment:
Důvod použití parametru bych opatřil dokumentací.

Reply:
Změnu jsem odstranil a aktualizoval phpDoc. Složka zasilkovna_src je vždy přítomna.